ArduCopter/RC_Channel.cpp

```diff
@@ -613,6 +629,23 @@ bool RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const AuxSwi
             }
             break;
 
+        case AUX_FUNC::TIE_DOWN_RELEASE:
```

Comment:
```suggestion
#if AP_AUX_FUNC_TIE_DOWN_RELEASE_ENABLED
        case AUX_FUNC::TIE_DOWN_RELEASE:
```

(and define in `libraries/RC_Channel/RC_Channel.h`)

This code should also be in `libraries/RC_Channel/RC_Channel.cpp` so all vehicles get it.